salt/utils/decorators/__init__.py

```diff
@@ -49,14 +53,36 @@ def test():
             @depends('modulename', fallback_function=function)
             def test():
                 return 'foo'
-        '''
 
+        .. code-block:: python
+
+        This can also be done with the retcode of a command, using the
+        `retcode` argument:
```

Comment:
This is _very_ picky, but 
```
`retcode`
```
Needs to be:
```
``retcode``
```
In order for the docs to render that as a code section. RST needs double backticks, while markdown only needs single backticks.

Reply:
Fair enough, though I'm pretty certain we don't build docs for this page. However, if we do start doing so, we should at least have it correct.